exploration_scripts/seven_segment_trajectory

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
```

Comment:
This file seems to have overlapping code with many other files -- was it a starter file that was intended to be replaced by better organized files?

Reply:
Oops! Yes. Deleted.